indico/modules/events/templates/emails/event_creation.txt

```diff
@@ -0,0 +1,48 @@
+{% extends 'emails/base.txt' %}
+
+{% block subject -%}
+    New {{ event.type }} in Indico
+{%- endblock %}
+
+{% block header -%}{%- endblock %}
+
+{% macro render_dates_and_access(event) %}
+    _Date_
+        {{ event.start_dt_local|format_datetime(timezone=event.timezone) }} -> {{ event.end_dt_local|format_datetime(timezone=event.timezone) }}
+    _Access_
```

Comment:
Why not Link or URL? "Access" doesn't make much sense.

We could also make this a bit more compact for occurrences:

```
_Other Occurrences_
    Jul 19 2016 - https://indico.cern.ch/event/1337
    Jul 20 2016 - https://indico.cern.ch/event/1338
```